src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -980,6 +980,41 @@ Example:
     def list(self):
       return {"msg": "Hello"}
 
+How to create a dedicated UI endpoint which uses the stable REST API?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes we want to combine multiple calls into one single call
```

Comment:
Maybe we should mention another good reason for using UI endpoints: Use UI endpoints if the result of the API request is only used and meant for usage in the UI.